ci/pipelines/deploy-to-test.yml

```diff
@@ -1137,7 +1140,7 @@ jobs:
       - put: alpine-ecr-registry-test
         params:
           image: image/image.tar
-          additional_tags: tags/tags
+          additional_tags: tags/all-release-tags
```

Comment:
I don't think the `tasks/parse-release-tag.yml` job has an `all-release-tags` output file yet - the previous PR added `all-candidate-tags`?